packages/tests/server/responseMeta.test.ts

```diff
@@ -3,12 +3,17 @@
 /* eslint-disable @typescript-eslint/no-unused-vars */
 import { routerToServerAndClientNew } from './___testHelpers';
 import { initTRPC } from '@trpc/server/src';
+import { IncomingMessage, ServerResponse } from 'http';
 import fetch from 'node-fetch';
 
 test('set custom headers in beforeEnd', async () => {
   const onError = jest.fn();
 
-  const t = initTRPC.context().create();
+  interface Context {
+    req: IncomingMessage;
+    res: ServerResponse<IncomingMessage>;
+  }
+  const t = initTRPC.context<Context>().create();
```

Comment:
A random bug popped up here b/c `Context` wasn't typed

So I guess this is a good change

Reply:
makes sense since we access ctx.res below? surprised it hasn't screamed before...